ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Initiator.java

```diff
@@ -112,6 +118,12 @@ public void run() {
                 continue;
               }
 
+              if (replIsCompactionDisabledForTable(t)) {
```

Comment:
replIsCompactionDisabledForTable is most likely return false. So, can we move this check after noAutoCompactSet check?

Reply:
i think it wont make much diff

Reply:
auto_compact flag remains forever unless someone changes it. So, if it is false, then additional check can be avoided in that case. But repl first inc flag is false most of the time. So, we end up with 2 checks always.

Reply:
done